charts/zigbee2mqtt/values.yaml

```
@@ -76,16 +81,48 @@ config:
     # MQTT server authentication, uncomment if required:
     # user: my_user
     # password: my_password
+    # client_id: my_id
+    # Alternatively, credentials may be put into a separate file, managed through a secret:
+    # password: '!secret password'
+
+    # Optional: Include device information to mqtt messages (default: false)
+    include_device_information: true
 
   # USB / Serial settings
   serial:
     # Location of your zigbee device
-    port: /dev/ttyACM0
+    port: /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2400981-if00
+
+    # Optional: adapter type, not needed unless you are experiencing problems (options: zstack, deconz)
+    # adapter: deconz
 
   advanced:
     network_key: GENERATE
     log_output:
       - console
+    log_level: info
+
+    # Optional: ZigBee channel, changing requires re-pairing of all devices. (Note: use a ZLL channel: 11, 15, 20, or 25 to avoid Problems)
+    channel: 11
```

```suggestion
    # channel: 11
```
I suggest commenting this out, since it's offered as an optional config parameter.